content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -345,7 +345,7 @@ Solana.
 To begin, make a new folder and install the relevant dependencies:
 
 ```bash
-npm i @solana/web3.js npm i @solana/web3.js npm i @solana-developers/helpers npm i @metaplex-foundation/mpl-token-metadata npm i @metaplex-foundation/umi-bundle-defaults npm i @metaplex-foundation/umi-uploader-irys npm i --save-dev esrun
+npm i @solana/web3.js@1 npm i @solana/web3.js@1 npm i @solana-developers/helpers npm i @metaplex-foundation/mpl-token-metadata npm i @metaplex-foundation/umi-bundle-defaults npm i @metaplex-foundation/umi-uploader-irys npm i --save-dev esrun
```

Comment:
this line is loaded with duplicate installs. it doesnt look like it was from your pr, buts lets fix it while we are here
```suggestion
npm install @solana/web3.js@1 @solana-developers/helpers@1 @metaplex-foundation/mpl-token-metadata @metaplex-foundation/umi-bundle-defaults @metaplex-foundation/umi-uploader-irys esrun
```

Reply:
note: I am also intentionally pinning the `@solana-developers/helpers` package since it uses v1. we are going to release an update version for v2